README.md

```diff
@@ -728,6 +728,30 @@ In either case:
       end
     ```
 
+* <a name="unless-with-comparator"></a>Avoid `unless` with comparators if you can use `if` with an opposing comparator.<sup>[[link](#unless-with-comparator)]</sup>
```

Comment:
I strongly think we should endeavor to do so. I don't really understand why we're making this part of our style guide. I can imagine others might wonder too. We're better off if people understand why they're doing something rather than just having it mandated upon them.